docker/0001-An-approximate-manylinux3.patch

```diff
@@ -23,8 +23,8 @@ index 56b42ff..73950b8 100755
  
 -docker/build_scripts/prefetch.sh openssl curl
 -docker build --rm -t quay.io/pypa/manylinux1_$PLATFORM:$TRAVIS_COMMIT -f docker/Dockerfile-$PLATFORM docker/
+-docker system prune -f
 +docker build --rm -t manylinux3_x86_64 -f docker/Dockerfile-$PLATFORM docker/
```

Comment:
@JanuszL Can you explain how is this change helping to prevent pruning other docker images?

Reply:
I removed "docker system prune -f", the rest was already there.